src/main/java/trello/ui/pages/BoardPage.java

```diff
@@ -168,11 +175,24 @@ public String getCardTitle(final String listTitle, final String cardTitle) {
         return title;
     }
 
+    /**
+     * Does click a card to be selected.
+     *
+     * @param cardTitle defines the card title that search.
+     */
+    public void selectedCard(final String cardTitle) {
+        for (int i = 0; i < cards.size(); i++) {
+            if (cards.get(i).getText().equals(cardTitle)) {
+                cards.get(i).click();
+            }
+        }
+    }
+
     /**
      * Wait until Page object is found.
```

Comment:
Please, change the verb in third person.

Reply:
Solved